recipe/meta.json

```diff
@@ -32,5 +32,5 @@
       "pyyaml =6.0.*"
     ]
   },
-  "version": "2.1.0"
+  "version": "2.1.1"
 }
```

Comment:
Updated, based on the contents of `meta.yaml` (see below), by running `make meta` in the `base` conda environment.